src/main/webapp/jsp/instructorCourses.jsp

```diff
@@ -1,247 +1,17 @@
-                            Course ID<span class="icon-sort unsorted"></span>
-                        </th>
-                        <th onclick="toggleSort(this,2);" id="button_sortid" class="button-sort-none">
-                            Course Name<span class="icon-sort unsorted"></span>
-                        </th>
-                        <th class="align-center no-print">Action(s)</th>
-                    </tr>
-                </thead>
-                <%  for (CourseAttributes course : data.archivedCourses) { 
-                        idx++;
-                %>
-                    <tr>
-                        <td id="courseid<%=idx%>"><%=sanitizeForHtml(course.id)%></td>
-                        <td id="coursename<%=idx%>"><%=sanitizeForHtml(course.name)%></td>
-                        <td class="align-center no-print">
-                            <a class="btn btn-default btn-xs" id="t_course_unarchive<%=idx%>"
-                                href="<%=data.getInstructorCourseArchiveLink(course.id, false, false)%>">
-                                Unarchive</a>
-                            <a class="btn btn-default btn-xs" id="t_course_delete<%=idx%>"
-                                href="<%=data.getInstructorCourseDeleteLink(course.id,false)%>"
-                                onclick="return toggleDeleteCourseConfirmation('<%=course.id%>');"
-                                data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.COURSE_DELETE%>"
-                                <%  InstructorAttributes instructor = data.instructors.get(course.id);
-                                    if (!instructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE)) { %>
-                                       disabled="disabled"
-                                <%  } %>
-                                >
-                                Delete</a>
-                        </td>
-                    </tr>
-                <%  }
-                    if (idx == -1) { // Print empty row
-                %>
-                    <tr>
-                        <td></td>
-                        <td></td>
-                        <td></td>
-                    </tr>
-                <% } %>
-            </table>
-            <br>
-            <br>
-            <br>
-            <br>
-        <% } %>
-    </div>
+</c:set>
 
-    <div id="frameBottom">
-        <jsp:include page="<%=Const.ViewURIs.FOOTER%>" />
-    </div>
-</body>
-</html>
\ No newline at end of file
+<ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Add New Course" jsIncludes="${jsIncludes}">
+	<course:addCoursePanel courseIdToShow="${data.courseIdToShow}" courseNameToShow="${data.courseNameToShow}" googleId="${data.account.googleId}"/>
+	<course:statusMessage/>
+	
+	<course:activeCoursesTable title="Active courses"/>
+	<course:archivedCoursesTable title="Archived courses"/>
```

Comment:
Is there a benefit behind passing "Active courses" or "Archived courses" as an attribute instead using them as constants in their respective tags?